variable/technology/technologies.yaml

```diff
@@ -1089,3 +1089,30 @@ Rate Automatic Frequency Restoration Reserve|Electricity|Wind|OffShore:
    description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
                 for a Wind OffShore power plant
    unit: '%'
+   
+# description of Demand flexibility potentials
+Number of Units|Electricity|Residential|Demand Flexibility:
+   description: Number of units of kind "simple Hydro reservoir "
+   unit: 
+   
+Maximum Active power|Electricity|Residential|Demand Flexibility:
+   description: Potential for demand response in a given region/time frame, 
+                expressed as % decrease in residential electricity demand
+   unit: %
```

Comment:
the %-character needs to be put in between quotes (see line 1091 above) for yaml files to be parsed correctly (as indicated by the failing tests)